packages/theme/src/fonts/index.ts

```diff
@@ -1,5 +1,65 @@
+/**
+ * Fonts
+ *
+ * {@link https://zeroheight.com/3ddd0f892/p/211297-typography/b/655ccf}
+ */
+
+const xLargeParagraph = {
+  fontSize: '2rem',
```

Comment:
Perf: can we create constants for these rem values since it's used multiple times here? E.g.,

```ts
const rem = {
  '2': '2rem',
  '2.6': '2.6rem',
}
```

Or you can create a util function that memoizes the value:

```ts
const rems: Record<string, sring> = {}

const rem = (value: number): string => rems[value] || (rems[value = `${number}rem`)
```

Reply:
Refactored using memoize approach